README.md

```diff
@@ -1,47 +1,73 @@
+* A web server that wraps the database in a REST API. A user should never directly interact with the database, but rather through the API.
+* A frontend web server, that provides a barebones graphical web user interface.
+* An infrastructure-as-code configuration for deploying these resources on Azure.
+* Tools for implementing your own "services", as DTBase calls them: Snippets of code that run periodically or on demand and interact with the backend API, such as data ingress functions or forecasting models.
+* A few example services:
+    * One to ingress weather data from OpenWeatherMap
+    * One for running the Arima time series forecasting model
+* Basic access control (user accounts and logins).
+* Backend API endpoints, and for some of these frontend pages for easier interaction, for adding/editing/deleting
+    * sensors, and inserting and retrieving data associated with sensors.
+    * locations, and associating sensors with locations.
+    * models, and inserting and retrieving data outputted by models. By this we mean simply "registering" a model with DTBase, not adding the code to actually run the model.
+    * users. Separating users with different rights (e.g. admins) is currently unimplemented.
+    * services. These are URLs that one can call to trigger a snippet of code to run, which may interact with the backend API. For instance ingress functions or models.
 
-The primary aim of DTBase is to provide a software package that developers can fork and use to deploy their own digital twin with minimal effort. This is an ambitious project as digital twins vary significantly and therefore DTBase needs to be both robust and flexible.
-DTBase consists of three main parts: A PostgreSQL database, a backend app for interacting with the database and a frontend app for visuals. The database, backend and frontend communicate via restful APIs, therefore allowing users of DTBase to choose which components of the code base they require for their personal usecase.
+Some things DTBase does not, currently, have:
+* Anything related to physical 3D structure of the objects that it is twinning. We have a very basic system for tracking locations in the twin, and for instance assigning sensors to locations, but this remains underdeveloped.
+* Comprehensive visualisation tools. The frontend has basic time series plotting capabilities, but that's it.
+* Sophisticated model orchestration. The services infrastructure can be used to store parameters for models and run them on demand, but the rest remains in development.
+* Anything for using the digital twin to control the real asset.
 
-We try to design the code base so as many of DTBase's features as possible can be deployed either locally or via Azure. There is no reason that DTBase couldn't be deployed on other cloud services, but Azure is what the developers have access to and therefore is the default service accomodated.
+DTBase may not be for you if
```

Comment:
Lets make headings like this a ### header

Reply:
Done